docs/DeveloperGuide.md

```diff
@@ -590,86 +590,92 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * 1b. The input data is invalid.
 
   * 1b1. System shows an error message.
-
-    Use case resumes at step 2.
+  
+    Use case ends.
 
 **Use case 2: Search for a guest**
 
 **MSS**
 
-1. User requests to search for a guest.
+1. User requests to search for a guest through any of the guest's details.
 2. System returns the guest.
 
    Use case ends.
 
 **Extensions**
-* 1a. The guest is does not exist in the system.
+* 1a. The guest's details does not exist in the system.
 
-  Use case ends.
+    * 1a1. System returns no guests.
+
+      Use case ends.
 
 * 1b. The search data is invalid.
 
     * 1b1. System shows an error message.
 
-      Use case resumes at step 2.
+      Use case ends.
 
 **Use case 3: Update a guest's details**
 
 **MSS**
 
 1. User <u>searches for guest (Use case 2)</u>.
 2. User requests to edit a guest's details.
-3. System updates the guest.
+3. System updates the guest's details.
 
    Use case ends.
 
 **Extensions**
-* 2a. The edit data is invalid.
+* 2a. The edit name already exists.
```

Comment:
Should we say "The new name provided already belongs to another existing guest"

Reply:
Agreed!